tests/components/zha/test_sensor.py

```diff
@@ -368,6 +368,7 @@ async def async_test_pi_heating_demand(hass, cluster, entity_id):
         "report_count",
         "read_plug",
         "unsupported_attrs",
+        "initial_sensor_state",
```

Comment:
Note on why this change was made:
The initial state for most sensors is `STATE_UNKNOWN`, as they get created, even if there's no attribute in the attribute cache. (Before this change, this was all this test was checking for.)

However, that's exactly what we want to avoid for our "summ received" entity. It no longer gets created when there's no value in the attribute cache.
So, for this test, we plug the attribute for `current_summ_received` with an initial state and expect the entity to also be that initial state.
(In reality, the attribute value would be read by `ZCL_INIT_ATTRS` if pairing a new device, thus causing the entity to create if the attribute is supported and it has a value.)

The test for the "ZHA devices list" further below now confirm that the entity is not created when no attribute is plugged (so no cached attribute value present), as the entries for "summ received are removed there.